ModList/UI/Markdig/RenderHelpers.cs

```diff
@@ -7,57 +7,79 @@ namespace IPA.ModList.BeatSaber.UI.Markdig
 {
     internal static class RenderHelpers
     {
-        public static EmphasisFlags GetEmphasisFlags(EmphasisInline em)
-            => em.DelimiterChar switch
-            {
-                '~' => em.DelimiterCount switch
-                {
-                    1 => EmphasisFlags.Underline,
-                    2 => EmphasisFlags.Strike,
-                    var i when i > 2 => EmphasisFlags.Underline | EmphasisFlags.Strike,
-                    _ => EmphasisFlags.None
-                },
-                var c when c == '*' || c == '_' => em.DelimiterCount switch
-                {
-                    1 => EmphasisFlags.Italic,
-                    2 => EmphasisFlags.Bold,
-                    var i when i > 2 => EmphasisFlags.Italic | EmphasisFlags.Bold,
-                    _ => EmphasisFlags.None
-                },
+        public static EmphasisFlags GetEmphasisFlags(EmphasisInline em) => em.DelimiterChar switch
+        {
+            '~' => em.DelimiterCount switch
+            {
+                1 => EmphasisFlags.Underline,
+                2 => EmphasisFlags.Strike,
+                var i when i > 2 => EmphasisFlags.Underline | EmphasisFlags.Strike,
+                _ => EmphasisFlags.None
+            },
+            var c when c == '*' || c == '_' => em.DelimiterCount switch
+            {
+                1 => EmphasisFlags.Italic,
+                2 => EmphasisFlags.Bold,
+                var i when i > 2 => EmphasisFlags.Italic | EmphasisFlags.Bold,
                 _ => EmphasisFlags.None
-            };
```

Comment:
For stuff like this, its more obvious to me that its an expression switch if the body is indented one more level and the expression body arrow is on the line below the declaration.